python/modules/IcePy/Util.cpp

```diff
@@ -269,226 +269,132 @@ IcePy::PyObjectHandle::release()
     return result;
 }
 
-IcePy::PyException::PyException()
-{
-    ex = PyErr_GetRaisedException();
-    if (ex)
-    {
-        PyObject* type = reinterpret_cast<PyObject*>(Py_TYPE(ex.get()));
-        Py_INCREF(type);
-        _type = type;
-        _tb = PyException_GetTraceback(ex.get());
-    }
-}
+IcePy::PyException::PyException() { ex = PyErr_GetRaisedException(); }
 
 IcePy::PyException::PyException(PyObject* raisedException)
 {
     Py_XINCREF(raisedException);
     ex = raisedException;
-    PyObject* type = reinterpret_cast<PyObject*>(Py_TYPE(raisedException));
-    Py_INCREF(type);
-    _type = type;
-    _tb = PyException_GetTraceback(raisedException);
 }
 
-void
-IcePy::PyException::raise()
+namespace IcePy
```

Comment:
Did you intend to use the anonymous namespace, and no reopen IcePy?

As far as I can tell, these functions are local to this file and not declared in the header.

Reply:
Yes